test/functional/ui/output_spec.lua

```diff
@@ -230,4 +231,26 @@ describe("shell command :!", function()
       ]])
     end)
   end)
+
+  if iswin() then
+    describe('powershell', function()
+      local screen
+      before_each(function()
+        clear()
```

Comment:
this is nested in a describe (line 15) which already defines before_each(), which will result in a redundant screen-setup + `clear()`. So this before_each() isn't necessary. This should work instead:


```lua
    if iswin() then
      it('supports literal strings', function()
        set_shell_powershell()
        feed([[:!'echo $env:USERPROFILE'<CR>]])
        screen:expect{any="echo %$env:USERPROFILE", timeout=5000}
        feed([[:!"echo $env:USERPROFILE"<CR>]])
        screen:expect{any='echo '..os.getenv('USERPROFILE'), timeout=5000}
      end)
    end
```

This is preferred, since redundant `clear()` is inefficient and provokes the `exit_event` race issue :)